typescript/infra/scripts/deploy.ts

```diff
@@ -254,7 +253,12 @@ async function main() {
     if (environment !== 'mainnet3') {
       throw new Error('CCIP is only supported on mainnet3');
     }
-    config = getCCIPDeployConfig(filteredTargetNetworks);
+    config = Object.fromEntries(
+      targetNetworks.map((origin) => [
```

Comment:
Was the use of `targetNetworks` instead of `filteredTargetNetworks` intentional?

Reply:
good catch - doesn't make a big difference to what ends up getting deployed with the current inputs but yeah meant to use filtered - will fix + undo the whitespace change